templates/field/field--node--oe-news-sources--oe-news--full.html.twig

```diff
@@ -0,0 +1,36 @@
+{#
+/**
+ * @file
+ * News sources field template.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+<h2 class="ecl-u-type-heading-2">{{ 'Sources'|t }}</h2>
+<div class="ecl-list">
+  {% for item in items %}
+    <article class="ecl-content-item ecl-u-d-s-flex ecl-u-pb-m">
+      <div class="ecl-u-flex-grow-1">
+        <div class="ecl-content-item__title ecl-u-type-heading-5 ecl-u-mb-xs ecl-u-mt-none">
+          {%- if item.content['#url'] %}
+            {% include '@ecl-twig/link' with {
+              link: {
+                type: 'standalone',
+                icon_position: 'after',
+                label: item.content['#title'],
+                path: item.content['#url']
+              },
+              icon: {
+                name: 'external',
+                size: 's',
+                path: ecl_icon_path
+              }
+            } %}
+          {% else %}
+            {{ item.content['#title'] }}
+          {%- endif -%}
+        </div>
+      </div>
+    </article>
+  {% endfor %}
+</div>
```

Comment:
Looks like it would be more suitable to reuse the `list_item` pattern like we did for related_links field.
See \oe_theme_preprocess_field function inside oe_theme.theme.

Reply:
Unfortunately, we can't, for now, use the `list_item` pattern according to fact that we can't control the link's icon. I think it would be the more preferable solution is anyway to use a pattern but with additional parameter for controlling the link's type (for adding external icon if needed).
I suppose it can be impelmented in follow-up ticket.